hrms/payroll/doctype/salary_component/salary_component.js

```diff
@@ -70,6 +77,39 @@ frappe.ui.form.on("Salary Component", {
 		}
 	},
 
+	add_update_structure_button: function (frm) {
+		for (const df of ["Condition", "Formula"]) {
+			frm.add_custom_button(
+				__("Sync {0}", [df]),
+				function () {
+					frappe.confirm(
+						__("Update {0} for all existing Salary Structures?", [df]),
```

Comment:
Can we also show the list of structures that will be updated so that the user can execute this without any doubts?

The following salary structures will be updated... Are you sure you want to proceed?